skimage/data/meson.build

```diff
@@ -37,6 +37,7 @@ py3.install_sources([
   'microaneurysms.png',
   'moon.png',
   'multipage.tif',
+  'multipage_rgb.tif',
```

Comment:
Off-topic: Should we eventually strip these from the wheel and make them download only? E.g. that's ~7 MiB of a ~14 MiB wheel currently.

Reply:
Happy to do that; we should register skimage-data for that purpose, and make it an (optional?) dependency of the package, and include it in the wheel builds.

Reply:
We could do a similar thing to split out the tests, if they take up a lot of space. They're pure Python, so no need to have them bundled in each binary wheel.

Reply:
Referencing https://github.com/scikit-image/scikit-image/issues/7470. It's very relevant to this.